scripts/jest/config.base.js

```diff
@@ -25,5 +25,5 @@ module.exports = {
   testSequencer: require.resolve('./jestSequencer'),
 
   // TODO: Upgrade to Jest 26 which uses jsdom 16 by default.
```

Comment:
remove this line?

Reply:
unfortunately that didn't fix the problem

Reply:
I just meant removing the todo comment